src/Microsoft.AspNetCore.Razor.Language/RazorEngine.cs

```diff
@@ -40,8 +40,25 @@ public static RazorEngine CreateDesignTime(Action<IRazorEngineBuilder> configure
 
         public static RazorEngine CreateEmpty(Action<IRazorEngineBuilder> configure)
         {
+            if (configure == null)
+            {
+                throw new ArgumentNullException(nameof(configure));
+            }
+
             var builder = new DefaultRazorEngineBuilder(designTime: false);
-            configure?.Invoke(builder);
+            configure.Invoke(builder);
+            return builder.Build();
+        }
+
+        public static RazorEngine CreateDesignTimeEmpty(Action<IRazorEngineBuilder> configure)
```

Comment:
Should have always had this since we have a runtime variant. Doesn't ever make sense to not provide a configure method to create empty because an empty razor engine doesn't function.